pravega/pravega
# Issue 6259: Large Appends in Client

**Change log description**  
Add client side support for large appends

**Purpose of the change**  
Fixes #6259

**What the code does**  
Implements large event support in the client

**How to verify it**  
TODO: We need to add an integration test to make sure this works with the server implementation.
